_data/ols-3-projects.yaml

```diff
@@ -0,0 +1,747 @@
+# List of projects for OLS-2
+#
+# Check OLS-1 for examples
+---
+- name: Practical Guide to Reproducibility in Bioinformatics
+  participants:
+    - sk-sahu
+  mentors:
+    - hrhotz
+  description: |
+    Reproducibility in life science always been in debate.
+    Althogly lately standard practices have adopted in different aspects,
+    sometimes these might not practically applicable by the end researcher.
+    Reproducibility practices need to be used by all, then only it will be
+    successful in both ends (research producer and research reader).
+    To bridge this gap, a lot of practical training is required.
+    This is where this project comes in picture.
+    Here we aim of creating training materials for doing
+    reproducible research in a more practical and application-oriented way
+    with a specific focus on niche areas of respective research
+    (for now bioinformatics as a starting point).
+    Not only limited to materials, as well as virtual training
+    by promoting the practuce of creating and sharing of Computational
+    Environment and Analysis in a reproducible manner with systems in scaling.
+
+- name: Mapping Science using Open Scholarship
+  participants:
```

Comment:
Do we want to copy ols-2 participants into ols-3? It might be clearer to leave this file blank or with one sample entry...

Reply:
I forgot to change that and I dreamed about it last night. I made the changes